.github/workflows/fast.yml

```diff
@@ -148,10 +148,16 @@ jobs:
       uses: ./.github/actions/setup
     - name: Add minimal no_std target
       run: rustup target add thumbv7em-none-eabihf
+    - name: Install ARM bare-metal toolchain (required for blst)
```

Comment:
Played around with trying to remove this via `portable` (#1495) but it didn't help at all.

Seems the root issue is that we need a cross-compiler for this target (from an x86 machine running these tests).